lmdeploy/model.py

```diff
@@ -593,6 +593,8 @@ def match(cls, model_path: str) -> Optional[str]:
             return 'internvl2-internlm2'
 
 
+@MODELS.register_module(name='internlm-xcomposer2d5')
+@MODELS.register_module(name='internlm-xcomposer2-4khd')
```

Comment:
They mean the same chat template. I think we can remove `internlm-xcomposer2-4khd`, can't we?

Reply:
May add UT for this chat template